src/Evolution/DgSubcell/Actions/TciAndRollback.hpp

```diff
@@ -45,10 +45,16 @@ namespace evolution::dg::subcell::Actions {
  * \brief Run the troubled-cell indicator on the candidate solution and perform
```

Comment:
This whole commit is a bug. The design was intentional because if you want to go to subcell on a boundary it's because the solution is bad so the simulation should die. The only reason it conflicts with "always use subecll" is because the BCs aren't implemented. This commit makes it so that BCs can't be supported for subcell, a major regression.